spec/features/sessions_spec.rb

```diff
@@ -56,6 +56,40 @@ def login
         expect(page).to have_content('Session timed out!')
       end
     end
+
+    describe 'return after timeout' do
+      let(:project) { create(:project) }
+
+      let(:submit_login_details) do
+        fill_in 'login', with: user.email
+        fill_in 'password', with: password
+        click_button 'Log in'
+      end
+
+      before do
+        login
+      end
+
+      it 'redirects to previous page' do
+        Timecop.freeze(Time.now + 1.hour) do
```

Comment:
Is there an alternative to using `Timecop.freeze`? Introducing more `#freeze` calls might cause issues in CI.